site/docs/about/glossary.mdx

```diff
@@ -36,3 +36,11 @@ Salt is the J.P. Morgan design system, an open-source solution for building exce
 Salt is under active development. At this stage, we don't recommend theming unless absolutely necessary.
 
 For an overview of theming, refer to our [theming documentation](/salt/themes/index).
+
+## Disabled state
+
+A component in a disabled state isn't considered relevant to the user’s workflow at that time. As the function isn't available to the user, the component doesn't need to meet minimum contrast requirements. Consider if a disabled state is the right UI pattern, or if a progressive reveal might be more appropriate. Salt doesn't offer error, warning, or success states for disabled components, as these imply required user action; consider using a read-only state or progressive reveal instead. Some components have exceptions, such as Button, so it can be discovered where context is important.
```

Comment:
```suggestion
A component in a disabled state isn't considered relevant to the user’s workflow at that time. As the function isn't available to the user, the component doesn't need to meet minimum contrast requirements. Consider if a disabled state is the right UI pattern, or if a progressive reveal might be more appropriate. Salt doesn't offer validation states (error, warning, or success) for disabled components, as these imply required user action; consider using a read-only state or progressive reveal instead. Some components have exceptions, such as Button, so it can be discovered where context is important.
```
Buttons have exceptions to what?